src/platform_impl/windows/dark_mode.rs

```diff
@@ -72,9 +74,12 @@ lazy_static! {
 
 /// Attempt to set dark mode on a window, if necessary.
 /// Returns true if dark mode was set, false if not.
-pub fn try_dark_mode(hwnd: HWND) -> bool {
+pub fn try_dark_mode(hwnd: HWND, preferred_theme: Option<Theme>) -> Theme {
```

Comment:
I'd actually rename this function, since it it's not about dark mode anymore, it's just about trying to get a theme, however when I see try, I'm expecting `Result`, so I'm not sure if this is a good name. Maybe `pick_theme` or something like that, if you have better ideas let me know.

Reply:
if your preferred theme is dark it could theoretically fail on old windows versions. In that case it would return `Theme::Light`.
So I think `try` should be included in the name. `try_select_theme`, `try_pick_theme` or `try_theme` 🤷 

I don't think it should necessarily return a result. I could change it to a result but it would make the calling code less nice IMO. (It would return a result, if the result is ok the current theme is the preferred theme if the result is a error than the theme is light. Which works but the caller of `try_dark_mode` needs to understand that the standard theme is light so I prefer to hide it away into the `try_` function)